### PR TITLE
zh: update faq from oncall issue

### PR DESCRIPTION
**Pull request by amyangfei**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-dm) that's required for repo owners to accept my contribution.

### What is changed, added, or deleted? (Required)

Update faq from oncall issue

### Which DM version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v3.0 changes)
- [x] v2.0 (TiDB DM 2.0 versions)
- [ ] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: zh/faq.md [hunk 1]
```diff
@@ -320,3 +320,11 @@ query-status test
     3. 重建任务并通过 `start-task task.yaml --remove-meta` 重新同步
     
 上述处理方案中，针对正常同步的 source（如上例 mysql2），重设增量任务时起始点需设置 `mysql-instances.meta` 为 `subTaskStatus.sync` 的 `syncerBinlog` 与 `syncerBinlogGtid`。
+
+## 在 DM 2.0 中开启 heartbeat，上游 MySQL 进行 vip 切换时，遇到 "heartbeat config is different from previous used: serverID not equal" 错误
```
Comment (lance6716) [inline zh/faq.md:324]: 我之前第一次见 Virtual IP 这个缩写，所以感觉写完整术语/全称更清晰一下

**Review by lance6716 — COMMENTED** [review]
rest lgtm

File: zh/faq.md (hunk 1 above)
Comment (amyangfei) [reply zh/faq.md]: updated

File: zh/faq.md [hunk 2]
```diff
@@ -320,3 +320,11 @@ query-status test
     3. 重建任务并通过 `start-task task.yaml --remove-meta` 重新同步
     
 上述处理方案中，针对正常同步的 source（如上例 mysql2），重设增量任务时起始点需设置 `mysql-instances.meta` 为 `subTaskStatus.sync` 的 `syncerBinlog` 与 `syncerBinlogGtid`。
+
+## 在 DM 2.0 中开启 heartbeat，虚拟 IP 环境下切换 DM-worker 与 MySQL 实例的连接，遇到 "heartbeat config is different from previous used: serverID not equal" 错误
+
+`heartbeat` 功能在 DM v2.0 已经默认关闭，如果用户在配置文件中开启会干扰高可用特性，在配置文件中关闭该项即可解决。步骤可参考 [AskTUG 该解决方案](https://asktug.com/t/topic/68351)。DM 将会在后续版本强制关闭该功能。
+
+## DM master 在重启后无法加入集群，报错信息为 "fail to start embed etcd, RawCause: member xxx has already been bootstrapped"
```
Comment (CharLotteiu) [inline zh/faq.md:328]:
```suggestion
## DM-master 在重启后无法加入集群，报错信息为 "fail to start embed etcd, RawCause: member xxx has already been bootstrapped"
```

File: zh/faq.md [hunk 3]
```diff
@@ -320,3 +320,11 @@ query-status test
...
+
+DM-master 会在启动时将 etcd 信息记录在当前目录，如果重启后当前目录发生变化，DM 缺失 etcd 信息从而启动失败。推荐使用 TiUP 运维 DM 避免这一问题。在需要使用二进制部署的场合，需要在 DM-master 配置文件中使用绝对路径配置 data-dir 项，或者注意运行命令的当前目录。
```
Comment (CharLotteiu) [inline zh/faq.md:330]:
```suggestion
DM-master 会在启动时将 etcd 信息记录在当前目录。如果重启后当前目录发生变化，会导致 DM 缺失 etcd 信息，从而启动失败。

推荐使用 TiUP 运维 DM 避免这一问题。在需要使用二进制部署的场合，需要在 DM-master 配置文件中使用绝对路径配置 data-dir 项，或者注意运行命令的当前目录。
```

File: zh/faq.md [hunk 4]
```diff
@@ -320,3 +320,11 @@ query-status test
     3. 重建任务并通过 `start-task task.yaml --remove-meta` 重新同步
     
 上述处理方案中，针对正常同步的 source（如上例 mysql2），重设增量任务时起始点需设置 `mysql-instances.meta` 为 `subTaskStatus.sync` 的 `syncerBinlog` 与 `syncerBinlogGtid`。
+
+## 在 DM 2.0 中开启 heartbeat，虚拟 IP 环境下切换 DM-worker 与 MySQL 实例的连接，遇到 "heartbeat config is different from previous used: serverID not equal" 错误
+
+`heartbeat` 功能在 DM v2.0 已经默认关闭，如果用户在配置文件中开启会干扰高可用特性，在配置文件中关闭该项即可解决。步骤可参考 [AskTUG 该解决方案](https://asktug.com/t/topic/68351)。DM 将会在后续版本强制关闭该功能。
```
Comment (CharLotteiu) [inline zh/faq.md:326]:
```suggestion
由于开启 `heartbeat` 功能会干扰 DM 高可用特性，在 v2.0 中已默认关闭。如果用户在配置文件中自行开启则会遇到此问题，在配置文件中关闭该项即可解决。步骤可参考 [AskTUG 该解决方案](https://asktug.com/t/topic/68351)。DM 将会在后续版本强制关闭该功能。
```
能否总结一下 AskTUG 里的解决步骤然后放到 FAQ 中来？页面跳转后的信息呈现是一个问答，解决步骤在最底部，对用户来说有很多无效信息，不够直观。同时能省去跳转这步也对用户更友好。

Comment (amyangfei) [reply zh/faq.md]: addressed, ptal

File: zh/faq.md [hunk 5]
```diff
@@ -320,3 +320,13 @@ query-status test
     3. 重建任务并通过 `start-task task.yaml --remove-meta` 重新同步
     
 上述处理方案中，针对正常同步的 source（如上例 mysql2），重设增量任务时起始点需设置 `mysql-instances.meta` 为 `subTaskStatus.sync` 的 `syncerBinlog` 与 `syncerBinlogGtid`。
+
+## 在 DM 2.0 中开启 heartbeat，虚拟 IP 环境下切换 DM-worker 与 MySQL 实例的连接，遇到 "heartbeat config is different from previous used: serverID not equal" 错误
+
+`heartbeat` 功能在 DM v2.0 已经默认关闭，如果用户在同步任务配置文件中开启会干扰高可用特性，在配置文件中关闭该项（通过设置 enable-heatbeat: false，然后更新任务配置）即可解决。DM 将会在后续版本强制关闭该功能。
```
Comment (yikeke) [inline zh/faq.md:326]:
```suggestion
`heartbeat` 功能在 DM v2.0 已经默认关闭，如果用户在同步任务配置文件中开启会干扰高可用特性，在配置文件中关闭该项（通过设置 `enable-heartbeat: false`，然后更新任务配置）即可解决。DM 将会在后续版本强制关闭该功能。
```

**Review by yikeke — APPROVED** [review]
Found a typo, rest LGTM

**Pull request merged**